# Generated by Django 5.1.6 on 2025-04-22 23:59

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ("authorization", "0001_initial"),
    ]

    operations = [
        migrations.RemoveField(
            model_name="auth",
            name="country",
        ),
        migrations.RemoveField(
            model_name="auth",
            name="img",
        ),
        migrations.RemoveField(
            model_name="auth",
            name="is_auth",
        ),
        migrations.RemoveField(
            model_name="auth",
            name="phone_number",
        ),
        migrations.AlterField(
            model_name="auth",
            name="username",
            field=models.CharField(
                error_messages={"unique": "A user with that username already exists."},
                help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
                max_length=150,
                unique=True,
                validators=[django.contrib.auth.validators.UnicodeUsernameValidator()],
                verbose_name="username",
            ),
        ),
    ]
